src/utils/buffer.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+/**
+ * This file exists to work around a number of issues with importing Buffer
+ * across Webpack, Node and Flow.
+ *
+ * Webpack (tested with 4.5.0) doesn't currently support use of Node globals,
+ * such as Buffer, in ES modules.
+ * See https://github.com/webpack/webpack/issues/7032
+ *
+ * Hence we want to import it explicitly e.g.
+ *
+ * import { Buffer } from 'buffer';
+ *
+ * But Node (tested with version 9.11.1 and --experimental-modules) only has
+ * Buffer as a property of the default export, not as a separate named export.
+ */
+
+import NodeBuffer from 'buffer';
+
+/**
+ * An alias for Node's Buffer class.
+ *
+ * Flow (tested with 0.69.0) knows about global.Buffer but not about
+ * require('buffer').Buffer.
+ * https://github.com/facebook/flow/issues/3723
+ */
```

Comment:
@motiz88 Flow `0.72` released and it includes your fix 🎉 Can you please update this PR?